src/api/Users.js

```diff
@@ -54,7 +54,12 @@ class Users extends Base {
         const accessToken: TokenLiteral = await TokenService.getReadToken(getTypedFileId(fileId), this.options.token);
 
         if (typeof accessToken === 'string') {
-            const url = `${this.getAvatarUrl(userId)}?access_token=${accessToken}`;
+            const options = {
+                access_token: accessToken,
+                pic_type: 'large',
+            };
+            const urlParams = Object.keys(options).map(key => key + '=' + options[key]).join('&');
```

Comment:
You can use `queryString.stringify` from the `query-string` module to build this string.